fix: typos in accessibility team pages (#571)
hms-dbmi/gehlenborglab-website: _teams/accessibility/research-in-biomedical-accessibility.md

## Evaluation of Accessibility in Life Sciences Data Resources

> Our lab has been conducting extensive research evaluating the accessibility of life sciences websites and data resources. In this ongoing project, we assessed over 3,000 data portals and journal websites using accessibility evaluation tools, like [WAVE](https://wave.webaim.org) and [Axe](https://www.deque.com/axe/). We also evaluated selected data portals with an actual screen reader user to identify real-world barriers for performing important tasks (e.g., data discovery).

Reviewer: ```suggestion
Our lab has been conducting extensive research evaluating the accessibility of life sciences websites and data resources. In this ongoing project, we assessed over 3,000 data portals and journal websites using accessibility evaluation tools, like [WAVE](https://wave.webaim.org) and [Axe](https://www.deque.com/axe/). We also evaluated selected data portals with a screen reader user to identify real-world barriers for performing important tasks (e.g., data discovery).
```

Consider amending the phrase "an actual screen reader user" to one of:
- "an experienced screen reader user" 
- "a screen reader user"
